Apsitvarkom.UnitTests/Api/Controllers/CleaningEventControllerTests.cs

```diff
@@ -222,16 +226,25 @@ public async Task GetById_RepositoryThrows_Status500InternalServerErrorReturned(
     [Test]
     public async Task Create_RepositoryInsertsOnce_CreatedAtActionResultReturned()
     {
+        var pollutedLocation = new PollutedLocation
+        {
```

Comment:
after the changes that I've suggested above (with the `ExistsByPropertyAsync) the creation of this will be no longer necessary, as we will check it only with a true or false.